base/stats.go

```diff
@@ -61,6 +61,28 @@ const (
 
 const StatsGroupKeySyncGateway = "syncgateway"
 
+const (
+	PrometheusValueTypeGauge   = "gauge"
+	PrometheusValueTypeCounter = "counter"
+
+	StatUnitNoUnits       = ""
+	StatUnitPercent       = "percent"
+	StatUnitBytes         = "bytes"
+	StatUnitNanoseconds   = "nanoseconds"
+	StatUnitSeconds       = "seconds"
+	StatUnitUnixTimestamp = "unix timestamp"
+
+	StatFormatInt      = "int"
+	StatFormatFloat    = "float"
+	StatFormatDuration = "duration"
+	StatFormatBool     = "bool"
+
+	StatVersionAdded3dot1dot0 = "3.1.0"
```

Comment:
Are there no `3.2.0` stats on `master`? That surprises me.

Reply:
Yes, talked about it, he's adding